packages/firebase_ai/firebase_ai/test/api_test.dart

```diff
@@ -998,6 +1022,118 @@ void main() {
         });
       });
 
+      group('UrlContextMetadata parsing', () {
```

Comment:
Let's add one more test case for multiple metadata in the urlContextMetadata's list.